test/e2e/framework/node/resource.go

```diff
@@ -336,7 +335,6 @@ func GetPublicIps(c clientset.Interface) ([]string, error) {
 // 2) Needs to be ready.
 // If EITHER 1 or 2 is not true, most tests will want to ignore the node entirely.
 // If there are no nodes that are both ready and schedulable, this will return an error.
-// TODO: remove references in framework/util.go.
```

Comment:
the function is used in https://github.com/kubernetes/kubernetes/blob/305377a2f577a47e4decc6ebfc4a3c2a393a00f5/test/e2e/framework/util.go#L1746
https://github.com/kubernetes/kubernetes/blob/305377a2f577a47e4decc6ebfc4a3c2a393a00f5/test/e2e/framework/util.go#L1775

it is not clear why this TODO exists and why this PR is removing it.

Reply:
66 matches in 46 files about this function. 
the reason for asking the owner 
/cc @jiatongw

Reply:
`GetReadySchedulableNodes` was originally defined in `framework/util.go`. To clean up the code in that file, this method was moved to package `node`.  But it was not deleted in `framework/util.go`, so that's why there is the comment. 

Now this method was deleted in `framework/util.go`, so I think it's safe to delete this comment too.